doc/source/install.rst

```diff
@@ -37,6 +37,6 @@ Mac
 
     xcode-select --install
     brew upgrade
-    brew install libxml2 libxmlsec1
+    brew install libxml2 libxmlsec1 pkg-config
```

Comment:
Please add  the same change to the README.rst

Reply:
And can you simplify commit message, I am not sure that backtrace should be in commit message.